unirec/unirec.h

```diff
@@ -764,6 +764,17 @@ ur_template_t *ur_create_template_from_ifc_spec(const char *ifc_data_fmt);
  */
 void ur_free_template(ur_template_t *tmplt);
 
+/** \brief Compares fields of two UniRec templates
+ * Function compares only sets of UniRec fields (direction is not compared).
+ * \note As it holds for UniRec templates in general, function returns invalid results if some
+ * UniRec field is redefined (i.e. undefined and defined as something else).
+ * \param[in] tmpltA Pointer to the first template.
+ * \param[in] tmpltB Pointer to the second template.
+ * \return If templates matches non-zero (1) value is returned, 0 othervise.
```

Comment:
I'd suggest:
```
Returns non-zero (1) value if templates matches. Otherwise, it returns 0.
```
It seems to be more readable.

Reply:
Ok.